roles/matterbridge/templates/matterbridge.toml

```diff
@@ -11,8 +11,8 @@ UseSASL=false
 
 Nick="{{ default_irc_bot_nick }}"
 #UseSASL=true
-#NickServNick=""
-#NickServPassword=""
+NickServNick="NickServ"
+NickServPassword="{{ default_irc_bot_nickserv_pass }}"
```

Comment:
Is this the secret that you need to provide to me or Tim? Or have we sorted the secret management? I see the `vault.yml` was updated in this PR too.

Reply:
all the relevant secrets are, i believe, in the updated vault yes

Reply:
Cool, then I am resolving this one.